quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/_1_4_x/lambdaTest.kt

```diff
@@ -0,0 +1,35 @@
+package co.paralleluniverse.kotlin.fibers.lang._1_4_x
+
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class lambdaTest {
```

Comment:
Do we actually check for instrumentation in any of these tests? Wouldn't they also pass if they're not instrumented and just not yield?

Reply:
Yes I agree, that is correct. They were swept up in this commit, hence the intriguing dinner1 method. But in another branch they are calling yield. I can bring across latest version into this commit.